pkg/snapstore/s3_snapstore.go

```diff
@@ -48,18 +48,20 @@ const (
 )
 
 type awsCredentials struct {
-	SSECustomerAlgorithm *string `json:"sseCustomerAlgorithm,omitempty"`
-	SSECustomerKey       *string `json:"sseCustomerKey,omitempty"`
-	Endpoint             *string `json:"endpoint,omitempty"`
-	S3ForcePathStyle     *bool   `json:"s3ForcePathStyle,omitempty"`
-	InsecureSkipVerify   *bool   `json:"insecureSkipVerify,omitempty"`
-	TrustedCaCert        *string `json:"trustedCaCert,omitempty"`
-	AccessKeyID          string  `json:"accessKeyID"`
-	Region               string  `json:"region"`
-	SecretAccessKey      string  `json:"secretAccessKey"`
-	BucketName           string  `json:"bucketName"`
-	RoleARN              string  `json:"roleARN"`
-	TokenPath            string  `json:"tokenPath"`
+	SSECustomerAlgorithm       *string `json:"sseCustomerAlgorithm,omitempty"`
+	SSECustomerKey             *string `json:"sseCustomerKey,omitempty"`
+	Endpoint                   *string `json:"endpoint,omitempty"`
+	S3ForcePathStyle           *bool   `json:"s3ForcePathStyle,omitempty"`
+	InsecureSkipVerify         *bool   `json:"insecureSkipVerify,omitempty"`
+	TrustedCaCert              *string `json:"trustedCaCert,omitempty"`
+	RequestChecksumCalculation *string `json:"requestChecksumCalculation,omitempty"`
+	ResponseChecksumValidation *string `json:"responseChecksumValidation,omitempty"`
```

Comment:
can you also please update the doc here: https://github.com/gardener/etcd-backup-restore/blob/master/docs/deployment/getting_started.md

Reply:
also update the example secrets here: https://github.com/gardener/etcd-backup-restore/blob/master/example/storage-provider-secrets/01-aws-s3-storage-secret.yaml